app/javascript/dashboard/helper/scriptGenerator.js

```diff
@@ -4,7 +4,7 @@ export const createMessengerScript = pageId => `
     FB.init({
       appId: "${window.chatwootConfig.fbAppId}",
       xfbml: true,
-      version: "v4.0"
+      version: "${window.chatwootConfig.fbApiVersion}"
```

Comment:
@tejaswinichile This change should not work as it renders as

`${window.chatwootConfig.fbApiVersion}` not `v12.0`.

The change should have been 
```
`${window.chatwootConfig.fbApiVersion}`
```

Reply:
Figured that the code is not used, hence removing it.

https://github.com/chatwoot/chatwoot/pull/4671